source/editableText.py

```diff
@@ -42,21 +43,30 @@ class EditableText(ScriptableObject):
 	#: Whether or not to announce text found before the caret on a new line (e.g. auto numbering)
 	announceNewLineText=True
 
-	def _hasCaretMoved(self, bookmark, retryInterval=0.01, timeout=0.03):
+	def _hasCaretMoved(self, bookmark, retryInterval=0.01, timeout=None):

```

Comment:
It's certainly a valid point, but I think it's a negligible/acceptable risk.

1. It's most likely that a caller would call this function either accepting the defaults or passing its own timeout, both of which will still work as expected.
2. This doesn't call super, so there's no chance that some method will receive `None` that wasn't expecting it.
3. The only risk I can think of is that a subclass might have hard-coded the old 0.03 default, in which case they won't get the user's configured value. However, at worst, this means some subclass won't behave consistently with the rest of NVDA, and this would be pretty clear from the debug log if caret movement started timing out. I also think the likelihood of someone having done this is unlikely.
4. Since it's a private method, maintaining backwards compat is a courtesy anyway, not an obligation. :)